sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -831,7 +868,7 @@ class CompactBlockProcessor internal constructor(
             }
 
             var start = syncRange.start
-            return mutableListOf<Batch>().apply {
+            return mutableListOf<BlockBatch>().apply {
```

Comment:
Could this be `buildList {}` instead of `mutableListOf().apply`?

Reply:
Good point, done.